hrui/js/src/civihr-popup/civihr-popup.js

```diff
@@ -12,6 +12,11 @@ Prevent Popups to overflow tables.
     if ($button.hasClass('btn-slide-active')) {
       $popup = $button.children('ul.panel');
 
+      // Use id of parent tr tag
+      $popup.attr("id", $button.closest('.crm-entity')[0].id);
+      // Use class of parent tr tag
+      $popup.addClass($($button.closest('.crm-entity')[0]).attr("class"));
```

Comment:
Anyway, if you do really need the class, then please cache `$button.closest('.crm-entity')[0]`, since you're using it twice
```js
var $button = $(this), $body = $('body'), $popup, $closestEntity, buttonOffset;

if ($button.hasClass('btn-slide-active')) {
  // ...
  var $closestEntity = $button.closest('.crm-entity')[0];

  $popup.attr("id", $closestEntity.id);
  $popup.addClass($closestEntity.attr("class"));

```